packages/blocks/src/store/reducer.js

```diff
@@ -36,7 +35,13 @@ export const DEFAULT_CATEGORIES = [
 	{ slug: 'reusable', title: __( 'Reusable blocks' ) },
 ];
 
-// Key block types by their name.
+/**
+ * Key block types by their name.
+ *
+ * @param { BlockType[]} types Block types of which to
```

Comment:
nitpick: I'm sure you'll find this later, but there's an extra space here

Reply:
Easy fix, thanks for pointing it out.